src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -478,17 +507,27 @@ def main():
     conf = None
 
     try:
+        # setup container.conf file for the callback server logs if needed
+        if loader_helper.is_config_modification_required():
+            if click.confirm(f"Set up a containers.conf file at {loader_common_names.CONTAINERS_CONF_PATH} if does not exist already?\n"
```

Comment:
```suggestion
            if click.confirm(f"The config file at {loader_common_names.CONTAINERS_CONF_PATH} "
                + "needs to be modified to allow for container logging. Do so now?\n"
```

Reply:
Updated!

Reply:
it looks like this part

>  f"Params 'seccomp_profile' and 'log_driver' will be added/updated under section [containers]"

Was removed as well, but that's good info for the user and should stay in

Reply:
Added back

Reply:
:+1: